addons/mail/models/mail_message.py

```diff
@@ -1004,16 +1008,17 @@ def _to_store(
             # model, res_id, record_name need to be kept for mobile app as iOS app cannot be updated
             data = message._read_format(fields, load=False)[0]
             record = record_by_message.get(message)
+            record_name = False
+            default_subject = False
             if record:
-                # sudo: if mentionned in a non accessible thread, user should be able to see the name
-                record_name = record.sudo().display_name
-                default_subject = record_name
-                if hasattr(record, "_message_compute_subject"):
-                    # sudo: if mentionned in a non accessible thread, user should be able to see the subject
-                    default_subject = record.sudo()._message_compute_subject()
-            else:
-                record_name = False
-                default_subject = False
+                with contextlib.suppress(MissingError):
```

Comment:
Why not also use `try … except` here?

Reply:
Linter warns me and suggest `contextlib.suppress` instead, probably because this is shorter for this pattern.